src/components/modals/TransferPDNTModal/TransferPDNTModal.tsx

```diff
@@ -49,6 +53,26 @@ function TransferPDNTModal({
       .then((res) => setState(res as PDNTContractJSON));
   }, [pdntId]);
 
+  useEffect(() => {
+    setAddressError(undefined);
+    if (toAddress.length < 43) {
+      return;
+    }
+    verifyAddress(toAddress);
+  }, [toAddress]);
+
+  async function verifyAddress(address: string) {
+    try {
+      setValidatingAddress(true);
+      await arweaveDataProvider.validateArweaveAddress(address);
+    } catch (error: any) {
+      setAddressError(error);
+      eventEmitter.emit('error', error);
+    } finally {
+      setValidatingAddress(false);
+    }
+  }
```

Comment:
why do we need this if the validation input does the validation?

Reply:
capturing the error for display

Reply:
we shouldn't need to do the call twice. use the validation callback to show the proper error message